src/board/boardConstants.js

```diff
@@ -0,0 +1,41 @@
+// index constants for properties 
+var MEDITERRANEAN_AVE = 0;
+var BALTIC_AVE = 1;
+var ORIENTAL_AVE = 2;
+var VERMONT_AVE = 3;
+var CONNECTICUT_AVE = 4;
+var ST_CHARLES_PLACE = 5;
+var STATES_AVE = 6;
+var VIRGINIA_AVE = 7;
+var ST_JAMES_PLACE = 8;
+var TENNESSEE_AVE = 9; 
+var NEW_YORK_AVE = 10;
+var KENTUCKY_AVE = 11;
+var INDIANA_AVE = 12;
+var ILLINOIS_AVE = 13;
+var ATLANTIC_AVE = 14;
+var VENTNOR_AVE = 15;
+var MARVIN_GARDENS = 16;
+var PACIFIC_AVE = 17;
+var NORTH_CAROLINA_AVE = 18;
+var PENNSYLVANIA_AVE = 19;
+var PARK_PLACE = 20;
+var BOARDWALK = 21;
+var READING_RR = 22;
+var PENN_RR = 23;
+var BO_RR = 24;
+var SHORTLINE_RR = 25;
+var ELECTRIC_CO = 26;
+var WATERWORKS = 27;
```

Comment:
Could we make these constants the actual number of their location on the board? I think they'd be useful for stuff like "Advance to St Charles Place"...